concrete/src/User/UserInfo.php

```diff
@@ -674,7 +707,7 @@ public function hasAvatar()
     }
 
```

Comment:
I think I misplaced this comment about deprecated methods: these deprecated methods are at the bottom of the file